code/modules/uplink/uplink_items.dm

```diff
@@ -103,7 +103,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 // Nuclear Operative (Special Offers)
 /datum/uplink_item/nukeoffer
 	category = "Special Offers"
-	surplus = 0
+	surplus = list(CARGO = 0, TRAITOR = 0)
```

Comment:
I dont know why we dont make the list of what we want and what probs dynamically at init.

Reply:
I'm not sure I understand what you're suggesting.